### ISSUE TITLE
CFclient no longer runs on VM 0.6

### ISSUE THREAD

**Issue by cstanke** [bitcraze/bitcraze-vm#26]
The master branch of crazyflie-clients-python (which ships on VM 0.6) now requires Python 3. While Python 3 is installed on the 0.6 VM, a [number of dependencies are not](https://github.com/bitcraze/bitcraze-vm/issues/22). The current user experience is that a fresh VM, where the [user has run "Update all projects](https://www.bitcraze.io/getting-started-with-the-crazyflie-2-0/)," will not run cfclient. Double-clicking the CFclient icon on the desktop gives no indication of the actual error (pyusb is missing).

**Comment by fredg02** [comment]
Hi,
You can either try to dist-upgrade to Xubuntu 15.04 or do the following in a terminal:

1. ```sudo mv /etc/apt/sources.list /etc/apt/sources.list.old```
2. ```sudo nano /etc/apt/sources.list```
3. in the editor paste the following:

    ```
    ## EOL upgrade sources.list
    # Required
    deb http://old-releases.ubuntu.com/ubuntu/ utopic main restricted universe multiverse
    deb http://old-releases.ubuntu.com/ubuntu/ utopic-updates main restricted universe multiverse
    deb http://old-releases.ubuntu.com/ubuntu/ utopic-security main restricted universe multiverse
    # Optional
    #deb http://old-releases.ubuntu.com/ubuntu/ utopic-backports main restricted universe multiverse
    ```
4. hit CTRL+O and Enter to save and exit
5. ```sudo apt-get update ```
6. ```sudo apt-get install python3-pip python3-pyqt4 python3-zmq```
7. ```sudo pip3 install pyusb==1.0.0b2```
8. ```sudo pip3 install pyqtgraph``` (this may take a little while)
9. right-click on the the "Crazyflie client" desktop shortcut, click on "Edit launcher", add ```python3``` at the beginning of the "command" field so it reads ```python3 /home/bitcraze/projects/crazyflie-clients-python/bin/cfclient ``` and click on save
10. double-click the "Crazyflie client" desktop shortcut and if a "Untrusted application launcher" warning message box appears click on "Mark Executable"

Hope that helps

Fred

**Comment by cstanke** [comment]
Thanks for the detailed workaround. I've upgraded my VM a while back (of course) and I generally do all my flying on my Mac (or Devo7e TX) anyway. I logged the bug because I was walking a new user through running on the VM when she hit "Update all projects" today and got stuck. I recognized many new users may be following the ['Getting Started' guide](https://www.bitcraze.io/getting-started-with-the-crazyflie-2-0/) and run across this same problem so I felt it should be tracked.

Hopefully your detailed and comprehensive instructions will help new users who run across this problem!

Regards!
-Chad

**Comment by krichardsson** [comment]
We messed up a bit with the release of the client, not being compatible with the VM, sorry about that! Thanks for the quick notification!

We have now updated the VM (excellent instructions @fredg02, thanks! ) and it is released as V0.8.
